README.md

```diff
@@ -199,20 +224,19 @@ from [bumblebench](https://github.com/AdoptOpenJDK/bumblebench) are executed and
 **NOTE** Future versions of this workflow will show the status of testing and meta data about how the binary was built.
 
 1. **Binaries are deployed** - Using the [OpenJDK Release Tool](https://ci.adoptopenjdk.net/view/Tooling/job/openjdk_release_tool/) (from the
-[openjdk-webiste-backend](https://github.com/AdoptOpenJDK/openjdk-website-backend) project) in order to:
-    1. deploy them to the various binary repos:
+[openjdk-website-backend](https://github.com/AdoptOpenJDK/openjdk-website-backend) project) in order to:
+    1. deploy them to the various binary repositories:
     ([openjdk8-binaries](https://github.com/AdoptOpenJDK/openjdk8-binaries/),
     [openjdk9-binaries](https://github.com/AdoptOpenJDK/openjdk9-binaries/),
     [openjdk10-binaries](https://github.com/AdoptOpenJDK/openjdk10-binaries/),
     [openjdk11-binaries](https://github.com/AdoptOpenJDK/openjdk11-binaries/),
     [openjdk12-binaries](https://github.com/AdoptOpenJDK/openjdk12-binaries/),
     [openjdk13-binaries](https://github.com/AdoptOpenJDK/openjdk13-binaries/),
+    [openjdk14-binaries](https://github.com/AdoptOpenJDK/openjdk14-binaries/),
+    [openjdk15-binaries](https://github.com/AdoptOpenJDK/openjdk15-binaries/),
     [openjdk-binaries](https://github.com/AdoptOpenJDK/openjdk-binaries/)).
 
-1. **Binaries made available** - Binaries are made available for download via the [website](https://www.adoptopenjdk.net)
```

Comment:
Should we add a section here mentioning installer/signing since we've got most of the rest?

Reply:
Added a TODO section for this